app/api/groups/get_granted_permissions.go

```diff
@@ -153,10 +153,10 @@ func (srv *Service) getGrantedPermissions(w http.ResponseWriter, r *http.Request
 			store.PermissionsGranted().PermissionIsAtLeastSQLExpr("grant_view", "enter")).
 		Select("DISTINCT item_id AS id")
 
-	managedGroupsQuery := store.ActiveGroupAncestors().ManagedByUser(user).
-		Group("groups_ancestors_active.child_group_id").
-		Having("MAX(can_grant_group_access)").
-		Select("groups_ancestors_active.child_group_id AS id")
+	// Used to be a subquery, but it failed with MySQL-8.0.26 due to obscure bugs introduced in this version.
+	// It works when doing the query first and using the result in the second query.
+	// See commit 1abc337a81f83462d4361b9876abee2a82c0e23a
```

Comment:
Isn't commit change its id with a rebase? I don't see it in this PR.

Reply:
fixed